docs/architecture/porting-existing-aspnet-apps/architectural-differences.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Architectural differences between ASP.NET MVC and ASP.NET Core
+description: Although at a high level ASP.NET MVC and ASP.NET Core share many concepts, there are many differences between them architecturally. Knowing these differences will help teams make informed decisions about how the will migrate their applications to .NET Core.
+author: ardalis
+ms.date: 11/13/2020
+---
+
+# Architectural differences between ASP.NET MVC and ASP.NET Core
```

Comment:
So that we're comparing apples to apples, should "ASP.NET Core" be "ASP.NET Core MVC"?